PACKAGING.md

```diff
@@ -4,36 +4,45 @@ Agama packages are available in the [YaST:Head:Agama project in
 OBS](https://build.opensuse.org/project/show/YaST:Head:Agama). This document summarizes the
 process we follow to build those packages.
 
-The process to build the Ruby-based ones (`rubygem-agama` and `rubygem-agama-cli`) is
-different from the one we use to build the web UI (`cockpit-agama`). The former packages are
-built in the same way that other YaST packages (through Rake tasks), while the latter package is
-automatically built in the Open Build Service.
+The process to build each package is slightly different depending on the technology we are using.
+While the Ruby-based one (`rubygem-agama`) is built as any other YaST package, the web UI
+(`cockpit-agama`) and the CLI (`agama-cli`) rely on [OBS source
+services](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.source_service.html).
 
-## Releasing a new version
+## Versioning policy
+
+We have decided to follow a single number schema: 1, 2, 3, etc. However, if we need to release a
+hot-fix, we might use a dotted version (e.g., 2.1). Moreover, all the components share the same
+version number. Releasing a new version implies that all of them get the new number, no matter if
+they contain changes or not.
+
+## Bumping the version
 
 In order to release a new version, we need to:
 
-* Update the version number in the `VERSION` file in the corresponding subdirectory. These files are
-  read by the Ruby-based packages when building the gems.
-* Tag the repository with the proper number. The process to build `cockpit-agama` uses this
-  information to infer the version. You can set the tag with something like:
+1. Update the version number in the `service/VERSION` file with the new number. These file is read
+  when building the `rubygem-agama` package.
+2. Run `bundle install` from the `service` directory.
+3. Add entries in the changes files.
+4. Open a pull request to get these changes into the repository.
+5. Once the pull request is merged, tag the repository with the proper version number. The processes
+   to build `cockpit-agama` and `agama-cli` use this information to infer the version. You can set
+   the tag with something like:
 
-      git tag --sign 0.5 --message "Version 0.5"
+      git tag --sign 1 --message "Version 1"
```

Comment:
```suggestion
      git tag --sign $(cat service/VERSION) --message "Version $(cat service/VERSION)"
```

I also see that we're tagging with a bare number. Even systemd which uses ordinal versioning pefixes the tags with a `v`, like `v256`. Just saying :)

Reply:
Good point. Actually, I think that the service is smart enough to handle the `v` prefix.

Reply:
I tried and it seems to work. So I have updated the instructions accordingly.